src/hooks/useToast.tsx

```diff
@@ -4,24 +4,22 @@ import {
   LinkProps,
   Text,
   ToastId,
-  UseToastOptions,
-  useToast as chakraUseToast,
   useColorModeValue,
+  useToast as chakraUseToast,
+  UseToastOptions,
 } from "@chakra-ui/react"
 import Button from "components/common/Button"
 import XLogo from "static/icons/x.svg"
 import { useRef } from "react"
 
 const useToast = (toastOptions?: UseToastOptions) => {
-  const useToastWithDefaults = chakraUseToast({
+  return chakraUseToast({
```

Comment:
We can then remove the `return` and the curly brackets too hehe

Reply:
true, I forgot to remove it